pull_secret.md

```diff
@@ -0,0 +1,30 @@
+# Updating the pull secret
+
+### Why do it
+
+When your Openshift cluster being provisioned, it uses a pull secret to identify you against many of the services offered by redhat such as Quay, Openshift, Registry.Redhat.io, etc, which is saved to the cluster after installation. To properly install the `trusted-artifact-signer` stack we have to add an entry for `registry.redhat.io`, to ensure who is using the service, and that they have the proper permissions for their ogranization. For any additional questions on this topic, refer to the [Openshift Documentation on using pull-secrets](https://docs.openshift.com/container-platform/4.13/openshift_images/managing_images/using-image-pull-secrets.html#images-update-global-pull-secret_using-image-pull-secrets).
+
+## Steps
+
+Firstly download the pull-secret data:
```

Comment:
`First, download the existing pull-secret from the openshift-config namespace`

(to clarify that they will be appending to the existing pull-secret)